### PR TITLE
FEATURE: Upgrade analytics.js to gtag.js

### PR DESCRIPTION
**Pull request by justindirose**
Per [Google](https://developers.google.com/analytics/devguides/collection/upgrade/analyticsjs), sites are encouraged to upgrade from `analytics.js` to `gtag.js` for Google Analytics tracking.

This PR makes this change for Discourse sites going forward. 

Note it's in draft status, so testing is not yet completed nor UI copy changes to coincide. I'd love some feedback on my approach first.

### CODE REVIEW

File: app/views/common/_google_universal_analytics.html.erb [hunk 1]
```diff
@@ -4,4 +4,6 @@
   auto_link_domains: SiteSetting.ga_universal_auto_link_domains
 } %>
 
+<script async src="https://www.googletagmanager.com/gtag/js?id=<%= SiteSetting.ga_universal_tracking_code %>"></script>
```
Comment (pmusaraj) [inline app/views/common/_google_universal_analytics.html.erb:7]: Are including the gtag script even if the site setting is empty?

Comment (justindirose) [reply app/views/common/_google_universal_analytics.html.erb:7]: No this partial gets loaded only if the site setting is present.

**Comment by justindirose** [comment]
@pmusaraj I made a couple additional changes (changed CSP URL for GA script with associated test, and copy). Mind having another review?

**Review by pmusaraj — APPROVED** [review]
👍

**Comment by justindirose** [comment]
Per Google, sites are encouraged to upgrade from `analytics.js` to `gtag.js` for Google Analytics tracking. This commit updates core Discourse to use the new `gtag.js` API Google is asking sites to use. This API has feature parity with `analytics.js` but does not use trackers.

**Pull request merged**